rdke-raspberrypi.xml

```diff
@@ -37,11 +38,12 @@
     <project groups="products" name="meta-product-raspberrypi" path="rdke/product/meta-raspberrypi4" remote="rdkcentral" revision="refs/tags/1.0.7">
         <annotation name="MANIFEST_EXPORT_PATH" value="MANIFEST_PATH_PRODUCT_LAYER"/>
     </project>
-    <project groups="layers" name="rdk/components/opensource/oe/meta-raspberrypi" path="rdke/vendor/meta-raspberrypi" remote="cmf" revision="dunfell">
-        <annotation name="MANIFEST_EXPORT_PATH" value="MANIFEST_PATH_BSP"/>
-    </project>
     <project groups="layers" name="meta-rdk-raspberrypi" path="rdke/vendor/meta-rdk-raspberrypi" remote="rdkcentral" revision="refs/tags/1.2.5">
         <annotation name="MANIFEST_EXPORT_PATH" value="MANIFEST_PATH_PLATFORM"/>
         <annotation name="MANIFEST_EXPORT_PATH1" value="MANIFEST_PATH_BBLAYERS_TEMPLATE"/>
     </project>
+    <!-- Use https://git.yoctoproject.org/meta-raspberrypi/commit/?h=dunfell&id=2081e1bb9a44025db7297bfd5d024977d42191ed -->
+    <project groups="vendor" name="meta-raspberrypi" path="rdke/vendor/meta-raspberrypi" remote="vendor" revision="2081e1bb9a44025db7297bfd5d024977d42191ed">
```

Comment:
Can we change `remote="vendor"` to `remote="yocto"`?
So that it is clear where it is pulling from.